README.md

```diff
@@ -119,6 +119,26 @@ The application also supports a quiet mode, which can be enabled by setting the
   -y, --yaml=<yamlFile>                  The path to the yaml file. (Not needed for the get commands)
 ```
 
+##### [HINT - DEFAULT VALUES FOR SOME PARAMS]
+
+Since the parameters `api`, `organization` and `space` are rarely changed, there is a property file 
+in the directory that defines default values \
+(see `cloud.foundry.cli/src/main/resources/cf_control.properties`).
+
+So the default-values are:
+```
+  -a=api.run.pivotal.io
+  -o=cloud.foundry.cli
+  -s=development
+```
+
+This reduces the number of program parameters to be called.
+However, if a value other than the default value is required, 
+this corresponding parameter can be set with its value in the program call.
+
+*The specification is:*\
```

Comment:
I think on GitHub you don't need these backslashes. "Standard" markdown IIRC also just uses two trailing spaces, backslashes are only supported in some extended dialects.

Reply:
Thanks, good to know. (: